components/Article/AuthorList/index.stories.tsx

```diff
@@ -0,0 +1,9 @@
+import AuthorList from '.';
+
+export default { component: AuthorList };
+
+export const Default = {
+  args: {
+    authors: ['shanpriyan', 'ovflowd', 'manishprivet', 'unavailable-author'],
```

Comment:
Can we put random people from GitHub instead of us? Just pick a few random users. I don't like that our names get listed just because we're the people doing this code.

Reply:
Shall I pick the random users from `nodejs.dev` learn section